gradoop-flink/src/main/java/org/gradoop/flink/model/impl/operators/sampling/functions/PageRankResultFilter.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright © 2014 - 2018 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradoop.flink.model.impl.operators.sampling.functions;
+
+import org.apache.flink.api.common.functions.FilterFunction;
+import org.gradoop.common.model.impl.pojo.Vertex;
+import org.gradoop.flink.model.impl.operators.sampling.PageRankSampling;
+
+/**
+ * Retains all vertices with a PageRank-score greater or equal/smaller than a given
+ * sampling threshold - depending on the Boolean set in {@code sampleGreaterThanThreshold}.
+ *
+ * @param <V> EPGM vertex type
+ */
+public class PageRankResultFilter<V extends Vertex> implements FilterFunction<V> {
```

Comment:
I think here is no Generic necessary. 
`public class PageRankResultVertexFilter implements FilterFunction<Vertex> {`